exporter/otlp/test/opentelemetry/exporter/otlp/exporter_test.rb

```diff
@@ -382,6 +382,17 @@
       _(result).must_equal(SUCCESS)
     end
 
+    it 'records metrics' do
```

Comment:
We could move `InMemoryMetricsReporter` from https://github.com/open-telemetry/opentelemetry-ruby/blob/2bcb4643b144300b0e18cba6ae61e93eb75bc5b0/exporter/zipkin/test/test_helper.rb#L43-L42 to `opentelemetry-test-helpers` if reviewers would like.